### PR TITLE
Colorize terminal output, if possible

### PR DESCRIPTION
**Pull request by carstene1ns**
Also, allow the browser to filter/colorize the console log.

Needs testing on:
- [x] Windows (debug build opens console)
- [x] Web
- [x] Linux
- ~~Wii (usbgecko, may be emulated by dolphin)~~
- ~~3DS (debug build or with 3dslink)~~
- ~~Switch (nxlink)~~

Obligatory screenshot:

### CODE REVIEW

**Comment by carstene1ns** [comment]
test this please.

**Comment by Ghabry** [comment]
Btw fmtlib also supports colours but it lacks all the Pty etc detection and only uses ansi escape codes

**Review by fdelapena — APPROVED** [review]
Works for me, thanks!

The emscripten console log levels worked, too: https://easyrpg.org/play/pr2608/

File: resources/unix/bash-completion/easyrpg-player [hunk 1]
```diff
@@ -72,7 +72,7 @@ _easyrpg-player ()
       return
       ;;
     # these have no argument and shall be used exclusively
-    -@(v|-version|h|-help))
+    -@(v|-version|h|-help|-no-log-color))
```
Comment (carstene1ns) [inline resources/unix/bash-completion/easyrpg-player:75]: This is obviously wrong... 😅
Will remove when I am at home later.

**Comment by carstene1ns** [comment]
After further thinking about it, it does not make sense to have this on non-desktop platforms, so these just get the current behaviour.

**Comment by fdelapena** [comment]
Captura de pantalla de 2021-08-03 12-57-35

It works as expected on Windows.

**Comment by carstene1ns** [comment]
Thank you, I guess this is ready then 👍

File: src/player.cpp [hunk 2]
```diff
@@ -150,6 +150,15 @@ void Player::Init(int argc, char *argv[]) {
 	// Must be called before the first call to Output
 	Graphics::Init();
 
+	// FIXME: actual command line parsing is too late for setting this,
+	// it should be refactored after release to not break things now
+	for (int i = 1; i < argc; ++i) {
+		if (strcmp(argv[i], "--no-log-color") == 0) {
+			Output::SetTermColor(false);
+			break;
```
Comment (Ghabry) [inline src/player.cpp:158]: this is really a general problem that should be addressed in a further PR.

This is also a problem for --help and --version: Before printing there stuff the console already contains "EasyRPG" and "CLI".

**Review by Ghabry — APPROVED** [review]
LGTM, Fedora will just be unhappy because we vendor too much stuff ;)

**Pull request merged**